tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -661,6 +675,86 @@ protected void EnableTelemetry(bool enabled = true, int? standaloneAgentPort = n
             Assert.Equal(expectedServiceVersion, span.Tags.GetValueOrDefault(Tags.Version));
         }
 
+        protected void SetupRcmConfiguration(string path = null)
+        {
+            if (path == null)
+            {
+                path = Path.GetTempFileName();
+                File.Delete(path);
+                Directory.CreateDirectory(path);
+                path = Path.Combine(path, "rcm_config.json");
+                if (File.Exists(path)) { File.Delete(path); }
+                _deleteRcmFile = true;
+            }
+            else
+            {
+                _deleteRcmFile = false;
+            }
+
+            SetEnvironmentVariable(ConfigurationKeys.Rcm.FilePath, path);
+        }
+
+        protected void CleanupRcmConfiguration()
+        {
+            if (EnvironmentHelper.CustomEnvironmentVariables.TryGetValue(ConfigurationKeys.Rcm.FilePath, out var rcmConfigPath))
+            {
+                if (File.Exists(rcmConfigPath) && _deleteRcmFile) { File.Delete(rcmConfigPath); }
+                EnvironmentHelper.CustomEnvironmentVariables.Remove(ConfigurationKeys.Rcm.FilePath);
```

Comment:
Not sure I understand, why do you need to remove this from `EnvironmentHelper`

Reply:
Extra actions to ensure no leaking of remote config from other tests

Reply:
But `EnvironmentHelper` is recreated for every test, no? 🤔 
And `CustomEnvironmentVariables` is in memory dictionary, so it teared down, on every test, too, I don't think this is required.

Reply:
You are right.